src/plots/cartesian/constants.js

```diff
@@ -68,6 +68,7 @@ module.exports = {
         'carpetlayer',
         'violinlayer',
         'boxlayer',
```

Comment:
Can `box` traces and `candlestick` coexist on the same subplot?

Reply:
good 👀 - they probably could have before this PR, but this morning they couldn't. Now they can https://github.com/plotly/plotly.js/pull/2561/commits/0b7541e34d63ce74cddb5d243a652b2df0430b70